lib/config.js

```diff
@@ -457,6 +464,7 @@ AWS.Config = AWS.util.inherit({
     sslEnabled: true,
     s3ForcePathStyle: false,
     s3BucketEndpoint: false,
+    s3DisableBodySigning: true,
```

Comment:
It might be good to default `s3DisableBodySigning` to true only when `sslEnabled` is true. If `sslEnabled` is explicitly set to false, then `s3DisableBodySigning` should default to false (and the documentation comments above should note this). Even though `shouldDisableBodySigning()` in the s3.js file checks that the protocol is https, this makes it clearer (and may make debugging easier). And if `{sslEnabled: false, s3DisableBodySigning: true}` is explicitly set, I would even add a validation listener to return an error when a request is being validated, to make it clearer that this combination is not allowed.